docs/source/science/NISAR/NISAR_access.ipynb

```diff
@@ -2,23 +2,23 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # ! pip install h5glance
</pre><p>Can probably be removed since it's listed above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I think I added this because it says glob cannot be found?</p>

Reply:
I'm guessing it's erroring out and not continuing the package installs - can probably remove glob from the installs or install glob2?

Reply:
<p>Looks good! I just replaced it was glob2. The only other one giving me issues is osgeo, it says there is a problem with the package and not pip. Says to use gdal instead?</p>